script.js

```diff
@@ -185,8 +185,8 @@ document.addEventListener('DOMContentLoaded', () => {
                           <div class="modal-description">
                             <div>${project.details}</div>
                             <div class="modal-buttons">
-                              <button class="modal-cta" type="button">See Live</button>
-                              <button class="modal-cta" type="button">See Source</button>
+                              <button class="modal-cta" type="button"><img id="seelive" src='./assets/seelive.png'/></button>
+                              <button class="modal-cta" type="button"><img id="seesource" src='./assets/seesource.png'/></button>
```

Comment:
- [ ] Your project is really good, just one thing to go. Kindly fix this issue on your mobile version, the button is out of the page. To match the Figma design.

![image](https://user-images.githubusercontent.com/35535130/174431912-f2160f7b-c2fa-4be6-8c28-0452b30f5fa8.png)

Reply:
Fixed and updated. 👍🏽